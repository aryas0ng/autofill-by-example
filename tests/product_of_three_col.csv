1,0,1,0
2,4,2,16
2,3,3,18
3,8,4,96
10,10,13,1300
5,5,1,
4,4,11,
19,4,13,
2,21,16,
1,21,16,
9,10,19,
7,7,2,
4,4,20,
3,3,3,
12,3,3,
4,16,3,
1,16,3,
26,27,4,
8,8,46,
3,3,5,
3,3,6,
8,11,6,
4,4,7,