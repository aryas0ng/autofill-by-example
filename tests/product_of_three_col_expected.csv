1,0,1,0
2,4,2,16
2,3,3,18
3,8,4,96
10,10,13,1300
5,5,1,25
4,4,11,176
19,4,13,988
2,21,16,672
1,21,16,336
9,10,19,1710
7,7,2,98
4,4,20,320
3,3,3,27
12,3,3,108
4,16,3,192
1,16,3,48
26,27,4,2808
8,8,46,2944
3,3,5,45
3,3,6,54
8,11,6,528
4,4,7,112